src/Config/GoogleAdsConfig.cs

```diff
@@ -153,24 +153,6 @@ public class GoogleAdsConfig : ConfigBase
         private ConfigSetting<long> clientCustomerId =
             new ConfigSetting<long>("ClientCustomerId", 0);
 
-        /// <summary>
-        /// The GMB login email.
-        /// </summary>
-        /// <remarks>
-        /// This setting is only for testing purposes.
-        /// </remarks>
-        private ConfigSetting<string> gmbLoginEmail =
-            new ConfigSetting<string>("GMBLoginEmail", "");
-
-        /// <summary>
-        /// Merchant Center ID to be used for Shopping campaigns.
-        /// </summary>
-        /// <remarks>
-        /// This setting is only for testing purposes.
-        /// </remarks>
-        private ConfigSetting<string> merchantCenterAccountId =
```

Comment:
Is this change intentional?

Reply:
Yeah, it is. It is an internal property that I had for testing code examples, but now that FT is used, we no longer need this property. Doesn't affect users because it is marked as internal and only for testing purposes.

Reply:
OK, thanks for confirming.